tests/unit/amo/pages/TestAddon.js

```diff
@@ -872,6 +872,27 @@ describe(__filename, () => {
     );
   });
 
+  it('hides developers comments if null', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      developer_comments: null,
+    });
+    const root = renderComponent({ addon });
+    expect(root.find('.Addon-developer-comments')).toHaveLength(0);
+  });
+
+  it('displays developers comments', () => {
+    const developerComments = 'some awesome developers comments';
+    const root = shallowRender({
+      addon: createInternalAddon({
+        ...fakeAddon,
+        developer_comments: developerComments,
+      }),
+    });
+    const devComments = root.find('.Addon-developer-comments div');
+    expect(devComments.html()).toContain(developerComments);
+  });
+
```

Comment:
It would be nice to have a test case similar to the one named `'allows some HTML tags in the description'`